schematic/models/GE_Helpers.py

```diff
@@ -374,6 +361,7 @@ def build_expectation_suite(
         suite_identifier = ExpectationSuiteIdentifier(
             expectation_suite_name=self.expectation_suite_name
         )
+        # TODO: Do we need to build data docs here during normal operation?
         self.context.build_data_docs(resource_identifiers=[suite_identifier])
```

Comment:
@andrewelamb @linglp @milen-sage - Is this code needed? It builds a bunch of HTML files, but I couldn't understand if they were actually used anywhere?

Reply:
Gotcha @GiaJordan What are your thoughts if I only call `self.context.build_data_docs(resource_identifiers=[suite_identifier])` if requested by the person running the code? This means when schematic is running as an API server it will never build these HTML files.

This is extra processing currently occurring in production schematic that is not needed. It's small, but non-zero.

On a side note: Do we have any environment variables/flags I could re-use that area already used in other areas of the code?

Reply:
@BryanFauble yeah that's fine. And there isn't one we've made that can be reused, but I think it would make sense to tie it to the verbosity requested for the command and include both lines like
```
if logger.isEnabledFor(logging.DEBUG):
    self.context.build_data_docs(resource_identifiers=[suite_identifier])
    #Webpage DataDocs opened here:
    self.context.open_data_docs(resource_identifier=suite_identifier)
```
It won't log to the logger, but `open_data_docs` will open the html file with the useful information in a browser and won't interrupt the rest of the run

Reply:
Perfect @GiaJordan I updated this code to:

```

        if logger.isEnabledFor(logging.DEBUG):
            self.context.build_data_docs(resource_identifiers=[suite_identifier])
            ##Webpage DataDocs opened here:
            # self.context.open_data_docs(resource_identifier=suite_identifier)
```

I am going to leave `open_data_docs` commented out like it was.